lib/c/strings.zig

```diff
@@ -0,0 +1,19 @@
+const std = @import("std");
+const common = @import("common.zig");
+
+comptime {
+    @export(&bzero, .{ .name = "bzero", .linkage = common.linkage, .visibility = common.visibility });
+}
+
+fn bzero(s: *anyopaque, n: usize) callconv(.c) void {
+    const s_cast = @as([*c]u8, @ptrCast(s));
```

Comment:
```suggestion
    const s_cast = @as([*]u8, @ptrCast(s));
```
`[*c]` should only ever appear in an automatically translated file.